src/librustdoc/html/render.rs

```diff
@@ -644,10 +644,9 @@ themePicker.onblur = handleThemeButtonsBlur;
     themes.appendChild(but);
 }});"#,
                  as_json(&themes));
-    write(cx.dst.join(&format!("theme{}.js", cx.shared.resource_suffix)),
-          theme_js.as_bytes()
-    )?;
-
+    write_minify(&cx.shared.fs, cx.path("theme.js"),
+                 &theme_js,
+                 options.enable_minification)?;
```

Comment:
Seems odd to minify more in a PR for "less minification"

Reply:
As a whole, it's less minification since most of minification is actually done in `search-index.js`.